pravega/flink-connectors
# [issue-199] Move documentation to connector repository

Signed-off-by: Vijay Srinivasaraghavan <vijayaraghavan.srinivasaraghavan@emc.com>

**Change log description**

- Created a documentation structure for connectors based on existing wiki content
- Fixed/Updated documentation based on latest changes to connector implementation
- Added `mkdocs` plugin support to generate web pages for using it in pravega.io site

**Purpose of the change**
To address https://github.com/pravega/flink-connectors/issues/199

**What the code does**
Flink connectors documentation changes

**How to verify it**
`./gradlew clean build` should build and test connectors properly.
`./gradlew clean mkdocsBuild` should build webpages for the documentation under build/mkdocs/documentation/generated directory location
